Sadot/User/UserHomePage.cs

```diff
@@ -49,16 +88,48 @@ private void FillTableList()
             tables = db.GetTablesData();
```

Comment:
Check tables != null

Reply:
the tables array is null at the first time and after the first time, I get new data from DB every time that this function calls even if were data in the tables array.
so both of the checks (== null / !=n null) will not work here.
what do you suggest to do?